README.adoc

```diff
@@ -10,77 +10,117 @@ image::https://github.com/rh-impact/pl-heudiconv/actions/workflows/ci.yml/badge.
+
+NOTE: The `pl-heudiconv` plugin does _not_ yet support the `--bids` argument, so you cannot currently use this plugin to convert the DICOM data to BIDS format.
 
 
 ## Installation
 
-`pl-heudiconv` is a link:https://chrisproject.org/[_ChRIS^] plugin_, meaning that it can run either from within _ChRIS_ or from the command-line.
+`pl-heudiconv` is a link:https://chrisproject.org/[_ChRIS_^] plugin, meaning that it can run either from within _ChRIS_ or from the command-line.
 
 image::https://ipfs.babymri.org/ipfs/QmaQM9dUAYFjLVn3PpNTrpbKVavvSTxNLE5BocRCW1UoXG/light.png[link=https://chrisstore.co/plugin/pl-heudiconv]
 
 
 ## Local Usage
 
-    docker run --rm quay.io/rh-impact/pl-heudiconv pl-heudiconv
+[subs=+quotes]
+....
+podman run --rm
+    -v _<path_to_input_dir_containing_DICOM_files>_:/incoming:z
+    -v _<path_to_output_dir>_:/outgoing:z
+    quay.io/rh-impact/pl-heudiconv pl-heudiconv
         [-h|--help]
         [--json] [--man] [--meta]
-        [--savejson <DIR>]
-        [-v|--verbosity <level>]
+        [--savejson _<DIR>_]
+        [-v|--verbosity _<level>_]
         [--version]
-        <inputDir> <outputDir>
+        /incoming /outgoing
+....
 
 
 ### Arguments
 
-    [-h] [--help]
-    If specified, show help message and exit.
+[subs=+quotes]
+....
+[-h] [--help]
+If specified, show help message and exit.
 
-    [--json]
-    If specified, show json representation of app and exit.
+[--json]
+If specified, show json representation of app and exit.
 
-    [--man]
-    If specified, print (this) man page and exit.
+[--man]
+If specified, print the man page and exit.
 
-    [--meta]
-    If specified, print plugin meta data and exit.
+[--meta]
+If specified, print plugin meta data and exit.
 
-    [--savejson <DIR>]
-    If specified, save json representation file to DIR and exit.
+[--savejson _<DIR>_]
+If specified, save json representation file to _DIR_ and exit.
 
-    [-v <level>] [--verbosity <level>]
-    Verbosity level for app. Not used currently.
+[-v _<level>_] [--verbosity _<level>_]
+Verbosity level for app. Not used currently.
 
-    [--version]
-    If specified, print version number and exit.
+[--version]
+If specified, print version number and exit.
+....
 
 
-Getting inline help is:
+To show inline help, run the following command:
 
-    docker run --rm quay.io/rh-impact/pl-heudiconv pl-heudiconv --man
+....
+podman run --rm quay.io/rh-impact/pl-heudiconv pl-heudiconv --man
+....
 
 ### Run
 
-You need to specify input and output directories using the `-v` flag to `docker run`.
+You must specify input and output directories using the `-v` flag in the `podman run` command, as follows:
 
-    docker run --rm -u $(id -u)                             \
-        -v $(pwd)/in:/incoming -v $(pwd)/out:/outgoing      \
-        quay.io/rh-impact/pl-heudiconv pl-heudiconv                        \
+[subs=+quotes]
```

Comment:
For my own learning of Asciidoc, what does this line mean? Do you happen to have a link to where I can read about it?

Reply:
It's a "substitution step that applies inline text formatting" -- I added it so that I could apply italics within a code block.
https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/